packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -181,13 +201,25 @@ class _InitializerState extends State<Initializer> {
     super.dispose();
   }
 
+  Future<void> _attemptUrlConnection() async {
+    final url = Uri.decodeFull(widget.url);
+    print('Decoding url: $url');
+    final bool connected = await FrameworkCore.initVmService(
+      '',
+      explicitUri: Uri.parse(url),
+      errorReporter: showErrorSnackBar(context),
+    );
+
+    if (!connected) {
+      _navigateToConnectPage();
+    }
+  }
```

Comment:
add a todo to show ui to navigate back to the connect page if the vm ever disconnects.
we don't want to just automatically navigate back to the connect page in that case because a user can be counting on still showing info from the disconnected page to file a bug, etc.

Reply:
Done.